src/compose/mod.rs

```diff
@@ -1499,16 +1534,7 @@ impl Composer {
                 }),
         );
 
-        let substituted_source = self
-            .sanitize_and_substitute_shader_string(source, &imports)
-            .map_err(|e| ComposerError {
-                inner: e,
-                source: ErrSource::Constructing {
-                    path: file_path.to_owned(),
-                    source: source.to_owned(),
-                    offset: 0,
-                },
-            })?;
+        let substituted_source = self.sanitize_and_set_auto_bindings(source);
```

Comment:
Is this being called twice (on the same `source` input) in the same function intentional?

Reply:
nope, removed, thanks.

i had some trouble with CI failures that didn't occur locally, i believe due to github treatment of `\r`s, so i moved the sanitize call up to the start of the function and forgot to remove the original call.